test/core/public/configure.js

```diff
@@ -908,4 +908,54 @@ describe('axe.configure', function() {
 			});
 		});
 	});
+
+	describe('given a standards object', function() {
```

Comment:
We should test that configuring something doesn't replace the other values on the standards object.